comp/forwarder/defaultforwarder/domain_forwarder.go

```diff
@@ -212,8 +212,10 @@ func (f *domainForwarder) Start() error {
 	// reset internal state to purge transactions from past starts
 	f.init()
 
+	maxConcurrentRequests := f.config.GetInt("forwarder_max_concurrent_requests")
```

Comment:
```suggestion
	maxConcurrentRequests := f.config.GetInt64("forwarder_max_concurrent_requests")
```

That way we can avoid the conversion on line 218 😄

Reply:
Nice!